composer/algorithms/mixup/mixup.py

```diff
@@ -28,25 +28,25 @@ def mixup_batch(input: torch.Tensor,
 
     This is done by taking a convex combination of ``input`` with a randomly
     permuted copy of ``input``. The permutation takes place along the sample
-    axis (dim 0).
+    axis (``dim=0``).
 
     The relative weight of the original ``input`` versus the permuted copy is
     defined by the ``mixing`` parameter. This parameter should be chosen
     from a ``Beta(alpha, alpha)`` distribution for some parameter ``alpha > 0``.
     Note that the same ``mixing`` is used for the whole batch.
 
     Args:
-        input (torch.Tensor): input tensor of shape ``(minibatch, ...)``, where
+        input (:class:`torch.Tensor`): input tensor of shape ``(minibatch, ...)``, where
```

Comment:
I dont think these are needed -- can you check the current docs @ docs.mosaicml.com to see if the Tensor is already linked?

Reply:
It looks like `Tensor` is already linked to me.

Reply:
@hanlint and @coryMosaicML I tried removing the :class: prefix and torch.Tensor was not linked upon rebuilding the documents. Am I missing something?

Reply:
Hmm, strange. That's fine to remove it here; I'll have to fix something in the sphinx `conf.py` later.